welcome/glossary.adoc

```diff
@@ -0,0 +1,257 @@
+In {product-title}, pods replace individual application containers as the smallest deployable unit.
+Pods are the orchestrated unit in {product-title}. {product-title} schedules and runs all containers in a pod on the same node. Complex applications are made up of multiple pods, each with their own containers. They interact externally and also with another inside the {product-title} environment.
+
+private registry::
+{product-title} can use any server implementing the container image registry API as a source of the image which helps the developers to push and pull their private container images.
+
+project::
+{product-title} uses projects to enable groups of users or developers to work together. A project defines the scope of resources, manages user access, and enforces resource quotas and limits.
+
+A project is a Kubernetes namespace with additional annotations that provide role-based access control (RBAC) and management capabilities. It serves as the central mechanism for organizing resources, ensuring isolation between different user groups.
+
+public registry::
+{product-title} can use any server implementing the container image registry API as a source of the image which allows the developers to push and pull their public container images.
+
+{op-system-base} {product-title} Cluster Manager::
+A managed service where you can install, modify, operate, and upgrade your {product-title} clusters.
+
+{op-system-base} Quay Container Registry::
+A Quay.io container registry that serves most of the container images and Operators to {product-title} clusters.
+
+replication controllers::
+An asset that indicates how many pod replicas are required to run at a time.
+
+ReplicaSet and ReplicationController::
+The Kubernetes `ReplicaSet` and `ReplicationController` objects ensure that the desired number of pod replicas are running at all times. If a pod fails, exits, or is deleted, these controllers automatically create new pods to maintain the specified replica count. Conversely, if there are more pods than required, the ReplicaSet or ReplicationController scales down by terminating excess pods to match the defined replica count.
+
+role-based access control (RBAC)::
+A key security control to ensure that cluster users and workloads have only access to resources required to execute their roles.
+
+route::
+A route is a way to expose a service by giving it an externally reachable hostname, such as www.example.com. Each route consists of a route name, a service selector, and optionally, a security configuration.
+
+router::
+A router processes defined routes and their associated service endpoints, enabling external clients to access applications. While deploying a multi-tier application in {product-title} is straightforward, external traffic cannot reach the application without the routing layer.
+
+scaling::
+The increasing or decreasing of resource capacity.
+
+service::
+A service exposes a running application on a set of pods.
+
+Service::
```

Comment:
Why is _service_ in here twice? If capital-S Service is different, _service_ in Line 225 needs to be capitalized.

Reply:
yes, you're right. I will remove the duplication.